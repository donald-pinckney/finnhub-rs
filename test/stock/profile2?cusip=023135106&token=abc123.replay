{"request":{"url":"https://finnhub.io/api/v1/stock/profile2?cusip=023135106&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/stock/profile2?cusip=023135106&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf74e6b614d-ORD","cf-request-id":"09a3846e890000614d1f3f3000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:50 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"max_age\":604800,\"report_to\":\"cf-nel\"}","report-to":"{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=%2BFEZvnHzh%2FlTBkawEogCnxdddQvt48tYHAiO4HBYJaXC2%2Bx0rmy86hu8FV2Jzga5YLiH2MIA59UKK9S0fpYer6puW%2Bjs6L%2FVRCzZVUt5X380%2FFESdovx\"}],\"group\":\"cf-nel\",\"max_age\":604800}","server":"cloudflare","set-cookie":"__cfduid=d0e3c69b7aacec8c47d95f56e94bda7391619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"52","x-ratelimit-reset":"1619234930"},"body":"eyJjb3VudHJ5IjoiVVMiLCJjdXJyZW5jeSI6IlVTRCIsImV4Y2hhbmdlIjoiTkFTREFRIE5NUyAtIEdMT0JBTCBNQVJLRVQiLCJmaW5uaHViSW5kdXN0cnkiOiJSZXRhaWwiLCJpcG8iOiIxOTk3LTA1LTE1IiwibG9nbyI6Imh0dHBzOi8vZmlubmh1Yi5pby9hcGkvbG9nbz9zeW1ib2w9QU1aTiIsIm1hcmtldENhcGl0YWxpemF0aW9uIjoxNjgxMDczLCJuYW1lIjoiQW1hem9uLmNvbSBJbmMiLCJwaG9uZSI6IjEyMDYyNjYxMDAwLjAiLCJzaGFyZU91dHN0YW5kaW5nIjo1MDQuMTE2NjMxLCJ0aWNrZXIiOiJBTVpOIiwid2VidXJsIjoiaHR0cDovL3d3dy5hbWF6b24uY29tLyJ9"},"format_version":3}