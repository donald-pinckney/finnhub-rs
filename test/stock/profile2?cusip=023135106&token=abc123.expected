CompanyProfile {
    country: "US",
    currency: "USD",
    exchange: "NASDAQ NMS - GLOBAL MARKET",
    ipo: "1997-05-15",
    name: "Amazon.com Inc",
    phone: "12062661000.0",
    shareOutstanding: 504.116631,
    ticker: "AMZN",
    weburl: "http://www.amazon.com/",
    logo: "https://finnhub.io/api/logo?symbol=AMZN",
    finnhubIndustry: "Retail",
}
